oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/agent/ClusterCachingAgent.groovy

```diff
@@ -301,6 +292,9 @@ class ClusterCachingAgent implements CachingAgent, OnDemandAgent, AccountAware,
       start = System.currentTimeMillis()
     }
 
+    // A non-null status indicates that the ASG is in the process of being destroyed (no sense indexing)
+    asgs = asgs.findAll { it.status == null }
```

Comment:
should this be `{ it.status != null }`? 
Or is `asgs` the collection of things to _not_ index?